sync/class.jetpack-sync-actions.php

```diff
@@ -296,23 +296,23 @@ static function maybe_schedule_sync_cron( $schedule, $hook ) {
 		}
 		$schedule = self::sanitize_filtered_sync_cron_schedule( $schedule );
 
+		// stagger the schedule time by some random fraction of the 2 x schedule interval (somewhat arbitrarily)
+		// so that on multisites with strong scheduling we don't DOS ourselves.
+		$schedule_time = time() + mt_rand( 0, 2*self::DEFAULT_SYNC_CRON_INTERVAL_VALUE );
```

Comment:
Took me a second to grok this. Basically, either schedule the event to start right away or up to `20` minutes from now.